blocks/property-listing/cards/cards.css

```diff
@@ -105,7 +105,7 @@
 .property-list-cards .property-labels .property-label {
   display: initial;
   height: 24px;
-  font-size: var(--body-font-size-xs);
+  font-size: var(--heading-font-size-xs);
```

Comment:
headings styles should only be used in Hx elements.

Reply:
Have updated this to the correct body font variable